### DISCUSSION TITLE
Need Help, Fatal Python error?

### DISCUSSION THREAD

**Discussion by cuteskater** [derekhe/msfs2020-google-map#129] [General]
Hello, my tread in the bug forum was closed :-( I still have a problem. I have tryed to resett my windows and installed python again. Nothing helped. I am getting this error

Fatal Python error: init_fs_encoding: failed to get the Python codec of the filesystem encoding
Python runtime state: core initialized
ModuleNotFoundError: No module named 'encodings'

Current thread 0x00001c40 (most recent call first):
<no Python frame>

What is it?

**Comment by drasticcode** [comment]
[Python error: String index out of range python
](https://drasticcode.com/string-index-out-range-python/
)
What is the “string index out of range” error, why it occurs, the solution to remove the string index out of range error

**Comment by Marakai** [comment]
This error can occur if your environment variables for the Python environment are wrong. You may not even need them! Did you install Python as admin or as non-admin (you can do both, even if your user account has local admin in Windows). Check your environment - both user and system - and see if `PYTHONHOME` or `PYTHONPATH` are set and if they are set correctly. You may even be able to go without them!
